### PR TITLE
Update UG for tag related commands

### PR DESCRIPTION
**Pull request by AriellaCallista**
Parts of UG updated:
- (new) create tags
- add 
- edit

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -84,21 +84,58 @@ Shows a message explaning how to access the help page.
...
 
+### Creating tags: `create`
+
+Creates a tag and categorises it to the specified category.
+You can customize these categories as per your needs and tags will be 
+color coded by the different categories. 
+
+The only restriction is you can only define up to 5 different categories.
+
+Format: `create t/CATEGORY NAME…​`
```
Comment (ketweeen) [inline docs/UserGuide.md:95]: I think doing `create t/CATEGORY TAG` or `create t/CATEGORY TAGNAME` would be clearer

Comment (ketweeen) [inline docs/UserGuide.md]: I think doing `create t/CATEGORY TAG` or `create t/CATEGORY TAGNAME` would be clearer

Comment (AriellaCallista) [reply docs/UserGuide.md]: noted!

**Review by ketweeen — APPROVED** [review]
LGTM

**Pull request merged**